charts/invenio/templates/_helpers.tpl

```diff
@@ -387,6 +387,73 @@ Add sentry environmental variables
 {{- end }}
 {{- end -}}
 
+{{/*
+Add S3 environmental variables if enabled.
+*/}}
+{{- define "invenio.config.s3" -}}
+{{- if or .Values.minio.enabled .Values.minioExternal -}}
```

Comment:
Either we use `.Values.invenio.s3FileStorage.enabled` or this, having both seems redundant.
If you prefer this approach, we can remove `invenio.s3FileStorage`, and I'd move this to a function in `_helpers.tpl` to use it to set the content of `INVENIO_FILES_REST_STORAGE_FACTORY` as well.